sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-webapp/README.md

```diff
@@ -68,7 +68,11 @@ azure:
     client-secret: <client-secret>
     tenant-id: <tenant-id>
     user-group:
-      allowed-groups: group1, group2
+      allowed-groups: group1, group2    # By default, groups name is used for permission control.
+#     enable-group-id: true             # If enable-group-id=true, allow-groups-id will be used for permission control. The allowed-groups will be invalidated.
+#     allowed-groups-id:       
```

Comment:
@backwind1233 what's your opinion of these properties?

Reply:
Actually, I think it's better to display groupId and groupname in pairs.
eg：
```
user-group:
  enable-group-id: true
  allowed-groups:
    - - groupName: groupName1
        groupId: groupId1
      - groupName: groupName2
        groupId: groupId2
```
@saragluna  @ZhuXiaoBing-cn